tests/foreman/api/test_environment.py

```diff
@@ -209,7 +209,6 @@ def test_positive_delete(self):
                     env.read()
 
 
-@skip_if_bug_open('bugzilla', 1262029)
```

Comment:
This is not the right way to skip an entire class. This actually messes up with the class and return a function instead.